Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/internal/Isotropic_remeshing/Uniform_sizing_field.h

```diff
@@ -74,7 +74,25 @@ class Uniform_sizing_field : public CGAL::Sizing_field<PolygonMesh>
     else
       return boost::none;
   }
+  
+  boost::optional<FT> is_protected_constraint_too_long(const halfedge_descriptor& h) const
+  {
+    const FT sqlen = 4./3. *sqlength(h);
+    if(sqlen > m_sq_long)
+      return sqlen;
+    else
+      return boost::none;
+  }
 
+  boost::optional<FT> is_protected_constraint_too_long(const vertex_descriptor& va,
+                                                       const vertex_descriptor& vb) const
+  {
+    const FT sqlen = 4./3. * sqlength(va, vb);
```

Comment:
4/3 is the factor for lengths, not for squared lengths, can you please fix that?